src/dsc/dsc.csproj

```diff
@@ -127,10 +123,7 @@
   </Target>
 
   <ItemGroup>
-    <None
-        Include="
-            $(MSBuildThisFileDirectory)..\..\NOTICE.txt;
-            $(MSBuildThisFileDirectory)..\..\LICENSE.txt">
+    <None Include="&#xD;&#xA;            $(MSBuildThisFileDirectory)..\..\NOTICE.txt;&#xD;&#xA;            $(MSBuildThisFileDirectory)..\..\LICENSE.txt">

```

Comment:
was this refactor intended?

Reply:
No, that was unintentionally.
It has been fixed. Good catch!